hasura/migrations/default/1665513818085_alter_table_public_users_add_column_entrance/up.sql

```diff
@@ -0,0 +1,2 @@
+alter table "public"."users" add column "entrance" varchar
+null;
```

Comment:
At the risk of increasing scope here significantly, we could use a [Hasura Enum](https://hasura.io/data-hub/data-models-and-authorization/schema-share-enum-reference-table/) instead of a string and then zeus would generate the enum for us, rather than us just enforcing it at the typescript layer manually.

This is just a suggestion if we have the appetite for changing the lift on this